elephant/neo_tools.py

```diff
@@ -45,17 +45,14 @@ def extract_neo_attrs(obj, parents=True, child_first=True,
 
     """
     attrs = obj.annotations.copy()
-    if not skip_array:
-        try:
-            for a in obj.array_annotations:
-                # Exclude labels and durations (and maybe other attributes) that are handled as array_annotations
-                # These would be duplicate
-                if a not in [_[0] for _ in obj._necessary_attrs + obj._recommended_attrs]:
-                    if "array_annotations" not in attrs:
-                        attrs["array_annotations"] = {}
-                    attrs["array_annotations"][a] = obj.array_annotations[a].copy()
-        except AttributeError:
-            pass
+    if not skip_array and hasattr(obj, "array_annotations"):
+        # Exclude labels and durations (and maybe other attributes) that are
+        # handled as array_annotations. These would be duplicate.
```

Comment:
[According to](https://github.com/NeuralEnsemble/elephant/issues/253#issuecomment-539903686) @apdavison, epoch labels and durations were not assimilated into array_annotations with Neo 0.8.0, so this commend should be updated.

Reply:
Updated the comment